lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -5355,14 +5355,17 @@ namespace Js
             return;
         }
         ForInCache * forInCacheArray = (ForInCache *)this->GetAuxPtr(AuxPointerType::ForInCacheArray);
-        if (isShutdown)
+        if (forInCacheArray)
```

Comment:
I think in most other places, we are explicit about saying `x != nullptr`. Same here?